src/java/com/wolfssl/provider/jsse/WolfSSLAuthStore.java

```diff
@@ -351,6 +363,33 @@ protected synchronized WolfSSLImplementSSLSession getSession(
         }
     }
 
+    /**
+     * Print summary of current SessionStore (LinkedHashMap) status.
+     * Prints out size of current SessionStore. If size is greater than zero,
+     * prins out host:port of all sessions stored in the store.
```

Comment:
typo 'prins'

Reply:
Thanks! Fixed and squashed into last commit.